package.json

```diff
@@ -23,6 +23,7 @@
   "license": "UNLICENSED",
   "devDependencies": {
     "markdown-proofing": "latest",
-    "shelljs": "^0.7.0"
+    "shelljs": "^0.7.0",
+    "elasticsearch": "^11.0.1"
```

Comment:
I'm not a `100%` sure if this should be a dev dependency. @kendaleiv or @comfroels will tell me.